Makefile

```diff
@@ -1,24 +1,22 @@
 NXDK_DIR = $(CURDIR)/../nxdk
 NXDK_NET = y
 
-XBE_TITLE = triangle
+XBE_TITLE = nxdk-rdt
 GEN_XISO = $(XBE_TITLE).iso
 SRCS = $(wildcard $(CURDIR)/*.c)
 SHADER_OBJS = ps.inl vs.inl
 
 SRCS += $(CURDIR)/lib/protobuf-c/protobuf-c.c
 
-CFLAGS_EXTRA += -I$(CURDIR)
-CFLAGS_EXTRA += -I$(CURDIR)/lib
+CFLAGS += -I$(CURDIR)
+CFLAGS += -I$(CURDIR)/lib
 
 include $(NXDK_DIR)/Makefile
 
-.PHONY:clean_local
-clean_local:
+.PHONY:clean-local
+clean-local: clean
```

Comment:
Old approach seems to work. Not sure why I thought it wasn't working. If it turns out to be a problem, there can be a followup PR.